src/common/ErrorMessages.cs

```diff
@@ -74,6 +74,7 @@ public static class ErrorMessages
         public const string IDX10229 = "IDX10229: TokenValidationParameters.TokenReplayCache was unable to add the securityToken: '{0}'.";
         public const string IDX10230 = "IDX10230: Lifetime validation failed. Delegate returned false, securitytoken: '{0}'.";
         public const string IDX10231 = "IDX10231: Audience validation failed. Delegate returned false, securitytoken: '{0}'.";
+        public const string IDX10232 = "IDX10232: validationParameters.CertificateValidator is null; User need to set CertificateValidator";
```

Comment:
I was adding tests and updated my commit few minutes ago. Please check the latest commit. Thanks.